package.json

```diff
@@ -22,9 +22,8 @@
   },
   "homepage": "https://github.com/UKHomeOffice/asl-public-api#readme",
   "dependencies": {
-    "@asl/migrator": "^1.6.0",
-    "@asl/mocks": "^2.2.0",
-    "@asl/schema": "^1.12.0",
+    "@asl/migrator": "^1.7.0",
+    "@asl/schema": "^1.13.2",
```

Comment:
@asl/mocks has been removed because it's already in the migrator as a dependency, and having multiple (potentially different) versions floating around is a recipe for WTFs.

Reply:
![](https://media1.tenor.com/images/19e59b5c8fbf858885d3939e05443681/tenor.gif?itemid=4180958)